CV Backend/src/main/java/com/qa/demo/domain/RoleManager.java

```diff
@@ -0,0 +1,49 @@
+package com.qa.demo.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RoleManager {
+    static private List<Role>roles = new ArrayList<Role>();
+    static RoleManager roleManagerRef;
+
+
+    private RoleManager()
+    {
+        addRole(new Role("admin", new Permissions()));
```

Comment:
shouldn't have the literal values like admin, needs to be an enum

Reply:
fixed using enum